simulariumio/constants.py

```diff
@@ -45,3 +45,18 @@ class DEFAULT_CAMERA_SETTINGS:
 
 
 DEFAULT_PLOT_MODE = "markers"
+
+
+class DISPLAY_TYPE:
+    SPHERE: str = "SPHERE"
+    CUBE: str = "CUBE"
+    GIZMO: str = "GIZMO"
+    PDB: str = "PDB"
+    OBJ: str = "OBJ"
+    FIBER: str = "FIBER"
```

Comment:
I would suggest removing CUBE and GIZMO at least until we actually support them (or e.g. comment them out and put a comment that says "coming up")